packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -804,23 +806,50 @@ class RawGestureDetector extends StatefulWidget {
   /// duplication of information.
   final bool excludeFromSemantics;
 
+  /// Describes the semantics notations that should be added to the underlying
+  /// render object [RenderSemanticsGestureHandler].
+  ///
+  /// It has no effect if [excludeFromSemantics] is true.
+  ///
+  /// When [semantics] is null, [RawGestureDetector] will fall back to a
+  /// default delegate which checks if the detector owns certain gesture
+  /// recognizers and calls their callbacks if they exist.
+  ///
+  ///  * During a semantic tap, it looks for [TapGestureRecognizer] and calls
+  ///    `onTapDown`, `onTapUp`, and `onTap`.
+  ///  * During a semantic long press, it looks for [LongPressGestureRecognizer]
+  ///    and calls `onLongPressStart`, `onLongPress`, `onLongPressEnd` and
+  ///    `onLongPressUp`.
+  ///  * During a semantic horizontal drag, it looks for [HorizontalDragGestureRecognizer]
+  ///    and calls `onDown`, `onStart`, `onUpdate` and `onEnd`. Then it looks
+  ///    for [PanGestureRecognizer] and calls the same callbacks.
+  ///  * During a semantic vertical drag, it looks for [VerticalDragGestureRecognizer]
+  ///    and calls `onDown`, `onStart`, `onUpdate` and `onEnd`. Then it looks
+  ///    for [PanGestureRecognizer] and calls the same callbacks.
+  final SemanticsGestureDelegate semantics;
```

Comment:
maybe add some sample code to show how to use this?

Reply:
Fixed